### PR TITLE
Produce nonshaded trino-jdbc

### PR DESCRIPTION
**Pull request by wendigo**
## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: client/trino-jdbc/pom.xml [hunk 1]
```diff
@@ -367,6 +367,21 @@
...
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>nonshaded</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <configuration>
+                            <classifier>nonshaded</classifier>
```
Comment (findepi) [inline client/trino-jdbc/pom.xml:380]: is it worth a code comment why this is useful?

**Review by findepi — APPROVED** [review]

File: client/trino-jdbc/pom.xml (hunk 1 above)
Comment (wendigo) [reply client/trino-jdbc/pom.xml:380]: Sure. I can add a comment to the commit and in the pom.

**Comment by mosabua** [comment]
Any issues with this @wendigo ? Do we need @electrum to look?

**Comment by mosabua** [comment]
Also .. should we document this for any reason? And also I assume the default jar is shaded and we dont want to break users and change that.. although arguably the default one should be non-shaded..

**Comment by gaeljw** [comment]
> Also .. should we document this for any reason?

I think this should absolutely be documented, maybe even with a word for users to know which dependency to use in which context and/or pros/cons of each dependency.

**Comment by wendigo** [comment]
@mosabua we need to follow up documenting this

**Comment by wendigo** [comment]
@gaeljw we will! Thanks for the feedback and sorry for shipping this so late

**Comment by gaeljw** [comment]
Thanks a lot folks :)

**Pull request merged**